CHANGES.md

```diff
@@ -1,3 +1,80 @@
+### NUnit 3.10 - March 12, 2018
+
+This release adds a .NET Standard 2.0 version of the framework which re-enables
+most of the features that have been missing in our earlier .NET Standard builds
+like parallelism, timeouts, directory and path based asserts, etc. It also contains
+numerous bug fixes and smaller enhancements. We've improved our XML docs,
+fixed performance issues and added more detail to Multiple Asserts.
+
+This release also contains source linked PDB files allowing developers to debug
+into the NUnit Framework. This allows you to track down errors or see how the
+framework works.
+
+In order to support the .NET Standard 2.0 version, the NUnit project switched to
+the new CSPROJ format and now requires Visual Studio 2017 to compile. This only
+effects people contributing to the project. NUnit still supports building and
+compiling your tests in older .NET IDE's and NUnit still supports older versions
```

Comment:
Double checked the rule. Usually we only use an apostrophe with an acronym if there is a period after every letter (I.D.E.'s) and otherwise only use an apostrophe after an acronym to make it possessive (IDE's); plural would just be IDEs. (English, right? 😐)

Reply:
I will update :stuck_out_tongue_winking_eye: